### PR TITLE
CV Submission for Bootcamp

### PR DESCRIPTION
**Pull request by darrenjia2004**
Here is my submission for the computer vision bootcamp. Let me know if there's anything I should change.

### CODE REVIEW

**Review by mikashaw — COMMENTED** [review]
Hey Darren, congrats on completing the bootcamp! would you mind just adding an accuracy plot and then I can approve/add feedback? thanks!

**Comment by darrenjia2004** [comment]
I've added an accuracy graph.

**Comment by aydanjiwani** [comment]
We typically set a threshold of 70% validation accuracy for the classifier. Try doing some research for ways to optimize your NN, adding more convolutional layers or changing the optimizer may be a good place to start. Also, I like the imgshow function, maybe try breaking up some of your training and plotting into functions as well?

**Comment by darrenjia2004** [comment]
I've added another convolutional layer and achieved 70% validation accuracy. The graphs and terminal output are updated.

**Review by mgupta27 — APPROVED** [review]
Looks good. I know that this PR is a litte outdated but if you are still interested in joining the team we have meetings every Monday. Hope to see you there!
